history.txt

```diff
@@ -1,6 +1,25 @@
 OMERO version history
 =====================
 
+5.4.0-m2 (July 2017)
+--------------------
+
+Developer preview release - **only intended as a developer preview for
```

Comment:
Should `only` be a new sentence, without the hyphen?  Since a new sentence is started within the bold text (`Use at…`)?